### ISSUE TITLE
cuDNN spatial log softmax and Chianti C++ library 

### ISSUE THREAD

**Issue by zhangxiaohuixuhao** [TobyPDE/FRRN#37]
Hi there,
      I'm really interested in using your network.
      However I get this error:
      2017-11-01 17:15:45,539 ERROR Cannot create cuDNN spatial log softmax. Install cuDNN and make sure that uses the GPU.
      2017-11-01 17:15:45,540 INFO Chianti C++ library is not available.Will default to slower Python implementation.
      I have updated theano and added " [lib]\n cnmem=1"to a .theanorc file in my home location.
      I really need your help,so I beg your reply.

**Comment by dingbaojin** [comment]
I have encountered the same error .
My cudnn version was 6, but  Theano can not support version 6,so I downgraded it from 6 to 5.It really works.

**Comment by huiyiygy** [comment]
Hi,  @dingbaojin 
I have downgraded cudnn from 5.1 to 5.0,but it still exists.
Here is my result:
2018-08-15 20:16:02,756 INFO Found supported Python version 2.7.
2018-08-15 20:16:02,804 INFO Successfully imported numpy.
2018-08-15 20:16:02,847 INFO Successfully imported cv2.
2018-08-15 20:16:02,881 INFO Successfully imported sklearn.
2018-08-15 20:16:02,976 INFO Successfully imported sklearn.metrics.
2018-08-15 20:16:02,976 INFO Successfully imported scipy.
Using gpu device 0: GeForce GTX 1080 (CNMeM is enabled with initial size: 90.0% of memory, cuDNN 5005)
2018-08-15 20:16:04,979 INFO Successfully imported theano.
2018-08-15 20:16:04,983 INFO Successfully imported lasagne.
2018-08-15 20:16:04,983 INFO Theano float is float32.
2018-08-15 20:16:04,983 ERROR Cannot create cuDNN spatial log softmax. Install cuDNN and make sure that theano uses the GPU.
2018-08-15 20:16:05,001 INFO Use Chianti C++ library.
2018-08-15 20:16:05,011 INFO Found CityScapes training set.
2018-08-15 20:16:05,011 INFO Found CityScapes validation set.

Wish your reply -_-

**Comment by a9311072** [comment]
Hi @huiyiygy 
I tried to follow the step to install the chianti, but it doesn't work.
could you share the steps for the **Chianti C++ library**?
Thanks.
